### PR TITLE
inputValue can't be 0 bug

### PR DESCRIPTION
**Pull request by CooperMoore9**

### CODE REVIEW

File: dist/uiUpdater.js [hunk 1]
```diff
@@ -115,10 +115,11 @@ function renameProject(project) {
     let projectTitle = document.querySelector(`.${project.name.replace(/\s/g, '').toLowerCase()}Title`);
     projectTitle.replaceWith(document.createElement('input'));
     let inputValue = document.querySelector('input');
+    console.log(inputValue);
```
Comment (Crowe7) [inline dist/uiUpdater.js:118]: console log

File: dist/uiUpdater.js [hunk 2]
```diff
@@ -115,10 +115,11 @@ function renameProject(project) {
...
     inputValue === null || inputValue === void 0 ? void 0 : inputValue.addEventListener('keypress', function (event) {
         if (event.key === 'Enter') {
-            if (checkNameProject(inputValue === null || inputValue === void 0 ? void 0 : inputValue.value) === true) {
-                alert("Projects can't be named the same");
+            if (checkNameProject(inputValue === null || inputValue === void 0 ? void 0 : inputValue.value) === true || (inputValue === null || inputValue === void 0 ? void 0 : inputValue.value.length) === 0) {
```
Comment (Crowe7) [inline dist/uiUpdater.js:121]: do you need the `inputValue === null || inputValue === void 0 ? void 0 :` here? can leave just the .length check?

File: src/uiUpdater.ts [hunk 3]
```diff
@@ -133,12 +133,13 @@ export function loopTasks(selectedProject: Project) {
         let projectTitle = document.querySelector(`.${project.name.replace(/\s/g, '').toLowerCase()}Title`) as Element
         projectTitle.replaceWith(document.createElement('input'))
         let inputValue = document.querySelector('input')
+        console.log(inputValue)
```
Comment (Crowe7) [inline src/uiUpdater.ts:136]: console log

**Review by Crowe7 — CHANGES_REQUESTED** [review]

File: dist/uiUpdater.js (hunk 2 above)
Comment (CooperMoore9) [reply dist/uiUpdater.js:121]: dist

**Pull request merged**